examples/arm/setup.sh

```diff
@@ -277,11 +279,39 @@ if [[ $is_script_sourced -eq 0 ]]; then
     # Create the setup_path.sh used to create the PATH variable for shell
     create_setup_path
 
-    # Setup the tosa_reference_model and dependencies
+    # Setup the TOSA reference model and serialization dependencies
     log_step "deps" "Installing TOSA reference model dependencies"
+    CMAKE_POLICY_VERSION_MINIMUM=3.5 \
+        pip install --no-dependencies -r "$et_dir/backends/arm/requirements-arm-tosa.txt"
+
+    pushd "$root_dir"
+    if [[ ! -d "tosa-tools" ]]; then
+        git clone https://git.gitlab.arm.com/tosa/tosa-tools.git
+    fi
+
+    pushd tosa-tools
+    git checkout 8468d041c50c6d806f3c1c18c66d7ef641e46580 # serialization lib pybindings
```

Comment:
Any impact on BC esp on Vela 4.4.[12]? I see the CI is OK so should be low risk.